contribute.md

```diff
@@ -109,6 +109,14 @@ git push origin develop
 
 Now follow step 4 to the end from above.
 
+### A note on Rubies
```

Comment:
Maybe this should go up at line 55 with the rest of the setup notes? I'd suggest 4 hashes as it's part of the setup notes rather than a new section in it's own right